react-cardknox-ifields/src/index.js

```diff
@@ -1,9 +1,11 @@
 
 import IField from "./iField";
 export default IField;
+export {default as CardknoxApplePay } from "./apple-pay";
+
 export {
     CARD_TYPE,
     CVV_TYPE,
     ACH_TYPE,
-    THREEDS_ENVIRONMENT
+    THREEDS_ENVIRONMENT,
```

Comment:
If you are exporting the apple pay constants for consumer to use, you need to export them here.
If not, they can be removed from the index.d.ts file

Reply:
Discussed